### ISSUE TITLE
Using php-sqllint as a Composer script raises RuntimeException, failing to require Console/CommandLine.php

### ISSUE THREAD

**Issue by JeanFred** [cweiske/php-sqllint#2]
Hello,

I would like to use php-sqllint in my continuous integration setup, as a Composer script. This fails as `Cli.php` cannot find `Console/CommandLine.php`

Steps to reproduce:
- Add the following `composer.json` to my project

```
{
    "require-dev": {
        "cweiske/php-sqllint": "^0.1.0"
    },
    "scripts": {
        "test": [
            "php-sqllint"
        ]
    }
}
```
- Run `composer install` 
- Run `composer test`
- Results

```
PHP Warning:  require_once(Console/CommandLine.php): failed to open stream: No such file or directory in /home/my-project/vendor/cweiske/php-sqllint/src/phpsqllint/Cli.php on line 16
PHP Fatal error:  require_once(): Failed opening required 'Console/CommandLine.php' (include_path='.:/usr/share/php:/usr/share/pear:/home/my-project/vendor/cweiske/php-sqllint/src/phpsqllint/../') in /home/my-project/vendor/cweiske/php-sqllint/src/phpsqllint/Cli.php on line 16
```

I imagine it is only the matter of a bit of autoload magic, but my expertise in PHP & Composer more or less stop there :)

**Comment by cweiske** [comment]
This is fixed with 0.1.1.

**Comment by JeanFred** [comment]
@cweiske Confirmed it is working! Thanks for fixing :)
